src/components/NavigationAppBar.js

```diff
@@ -119,26 +134,60 @@ class NavigationAppBar extends React.Component {
             </Typography>
             <Grow />
             <SectionDesktop>
-              {/* TODO add highlight for selected page */}
-              {/* TODO add tooltip for each menu item */}
-              <IconButton color="inherit" onClick={() => onChangePage(LANDING)}>
-                <ArrowForwardIcon />
-              </IconButton>
-              <IconButton color="inherit" onClick={() => onChangePage(INFO)}>
-                <InfoIcon />
-              </IconButton>
-              <IconButton color="inherit" onClick={() => onChangePage(IMPORT)}>
-                <InputIcon />
-              </IconButton>
-              <IconButton color="inherit" onClick={() => onChangePage(CATEGORIZE)}>
-                <CategoryIcon />
-              </IconButton>
-              <IconButton color="inherit" onClick={() => onChangePage(VISUALIZE)}>
-                <ShowChartIcon />
-              </IconButton>
-              <IconButton color="inherit" onClick={() => onChangePage(EXPORT)}>
-                <PublishIcon />
-              </IconButton>
+              <Tooltip title={<FormattedMessage id="navigation.start" />}>
+                <IconButton
+                  color="inherit"
+                  onClick={() => onChangePage(LANDING)}
+                  className={activePage === LANDING ? classes.active : ''}
+                >
+                  <ArrowForwardIcon />
+                </IconButton>
+              </Tooltip>
+              <Tooltip title={<FormattedMessage id="navigation.info" />}>
+                <IconButton
+                  color="inherit"
+                  onClick={() => onChangePage(INFO)}
+                  className={activePage === INFO ? classes.active : ''}
+                >
+                  <InfoIcon />
+                </IconButton>
+              </Tooltip>
```

Comment:
Would like to have each Desktop item as a custom DesktopMenuItem component, the same way Ahmed suggested with the MobileMenuItem.